test/driver_test.dart

```diff
@@ -247,10 +246,10 @@ class MockWorker implements Process {
   @override
   IOSink get stdin {
     _stdin ??= IOSink(_stdinController.sink);
-    return _stdin;
+    return _stdin!;
   }
 
-  IOSink _stdin;
+  IOSink? _stdin;
```

Comment:
I think this could be:

```suggestion
  late final IOSink _stdin = IOSink(_stdinController.sink);
```

And then the `stdin` getter can just be `=> _stdin;`

Reply:
Done, I simply removed stdin => _stdin and created `@override late final IOSink stdin = ...`